src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/ResponseHandlers.scala

```diff
@@ -0,0 +1,28 @@
+package it.pagopa.interop.backendforfrontend.api.impl
```

Comment:
The BFF does not handle specific errors, but just bubbles everything up.
We have a different task to implement specific error handling in this service.

You can just use the existing generic `handleError` that you may find in other endpoints, and delete this file